cryptography/src/bls12381/mod.rs

```diff
@@ -59,7 +59,7 @@
 //! conc=8 n=500 t=167     time:   [232.44 ms 238.31 ms 247.56 ms]
 //! ```
 //!
-//! ## Partial Signature Aggregation
+//! ## Threshold Signature Recovery
```

Comment:
I decided to wait to post other benchmarks with the expectation we'll shortly be migrating to the benchmarks page (introduced by #286)